### DISCUSSION TITLE
Localization doesn't seem to be working

### DISCUSSION THREAD

**Discussion by mouseman85** [microsoft-search/pnp-modern-search#2016] [Q&A]
Hi,

I'm having some issues with the localization when using the card view it shows the correct text but also the ID and I can't seem to figure out why. I have enabled localization which I thought would remove this?

![image](https://user-images.githubusercontent.com/11850876/163135940-59669b0f-28dc-4ee9-aae3-51e6dc64cdde.png)

**Comment by wobba** [comment]
Did you read and follow directions from the localization settings at https://microsoft-search.github.io/pnp-modern-search/usage/search-results/data-sources/#source-configuration?
